tools/cldr-apps/src/main/webapp/js/survey.js

```diff
@@ -4360,5 +4212,4 @@ function showUserActivity(list, tableRef) {
 					});
 				});
 			});
-	});
 }
```

Comment:
Indentation is very screwed up -- I left it this way so diff is easier -- but let's format all the js files soon, maybe with "npx prettier" default options